### PR TITLE
Add support for watchOS 3, drop support for iOS 7

### PR DESCRIPTION
**Pull request by designatednerd**
Heads up: This is part 1 of ??? to add watchOS support to Vokoder, which depends on the core of `VOKUtilites`. 

In order to test this, I added some stuff from each of the core dependencies to a sample watchOS app and made sure it all compiled and didn't barf when it ran. There's not really a good way to unit test watchOS stuff, but since it will be compiled automatically whenever the tests on the sample iOS app run, that should theoretically catch any regressions. 

I also dropped iOS 7 support since Xcode 8.x no longer supports building for anything below iOS 8. I consequently bumped to a new major version. I also plan to make this change in Vokoder, so @vokal-isaac now would be a good time to complain if you're against it.

### CODE REVIEW

**Comment by designatednerd** [comment]
@vokal-isaac @bryanluby @lindelldigital and/or @chillpop, thoughts?

**Comment by vokal-isaac** [comment]
Are you saying it's not possible to set a deployment target below iOS 8 on Xcode 8?

**Comment by joevandeventer** [comment]
I don't know if it's any more complicated than this, but, 

screen shot 2017-03-15 at 1 12 26 pm

Why my install of San Francisco as a system font is missing the bullet character is a discussion for another time.

**Comment by designatednerd** [comment]
This is the warning you get when you go to upgrade: 

screen shot 2017-03-15 at 1 17 36 pm

I suspect you could do it manually, but I don't believe it'll actually archive for anything older than 8.

**Comment by bryanluby** [comment]
Since this is now Xcode 8, do we also need to update the Travis `osx_image` value?

**Comment by designatednerd** [comment]
@bryanluby it was already set to osx_image: xcode8.2

**Comment by bryanluby** [comment]
@designatednerd 👍 I was looking at an old version of the file

**Comment by vokal-isaac** [comment]
I don't have time to look over the whole diff at the moment.  Here are my comments thus far:

- Since Xcode 8.whatever doesn't support a deployment target below iOS 8, upping the minimum deployment target to iOS 8 is fine.
- There's a commit message that says "drop iOS 8 support"—should that have been iOS 7 support?
- [Semantic Versioning says](http://semver.org/#spec-item-4) that 0.x / pre-1.x doesn't make any promises about what breaks when, so my inclination is to go with 0.11.x rather than crossing the 1.0 threshold.

**Comment by designatednerd** [comment]
@vokal-isaac yes that was a mistake, dropped 7 to 8, not dropped 8

I'm fine with whatever y'all want on the versioning, this just seemed like a big enough change to bump a major version. However, I shall abide by the selections of the crew

**Comment by vokal-isaac** [comment]
If we were post-0.x, I'd agree that it warranted a major version bump.  Since we're still in 0.x, I'm not sure we want to leave 0.x.  (Because 0.x is special.)

**Comment by bryanluby** [comment]
The `0.x`versioning is fine with me

File: Example/Pods/Manifest.lock [hunk 1]
```diff
@@ -40,8 +40,8 @@ EXTERNAL SOURCES:
...
 SPEC CHECKSUMS:
   KIF: 082eb65279e51c3092923802849eb796a04982ab
-  VOKUtilities: d9d47dc173f067b0c539c5b002cb442c12cfac4d
+  VOKUtilities: 9940668fd0816e728f959666ad84f028b237bc35
 
-PODFILE CHECKSUM: 9ccc53b03f9e3a9710f9636b4227ff058e608662
+PODFILE CHECKSUM: b4c75412af7a62397b6f2a8345de8b99a9100467
 
-COCOAPODS: 1.1.1
+COCOAPODS: 1.2.0.rc.1
```
Comment (chillpop) [inline Example/Pods/Manifest.lock]: want to use the release version of 1.2.0 ?

File: VOKUtilities.podspec [hunk 2]
```diff
@@ -36,25 +37,25 @@ Pod::Spec.new do |s|
   
   #UIKit specs:
   s.subspec 'UIColor+VOKAL' do |ss|
```
Comment (chillpop) [inline VOKUtilities.podspec:39]: I believe UIColor is available on watchOS too

Comment (designatednerd) [reply VOKUtilities.podspec:39]: You are correct

**Review by bryanluby — APPROVED** [review]

**Comment by designatednerd** [comment]
@vokal-isaac @lindelldigital @chillpop  light is green, trap is clean. thoughts?

**Review by chillpop — APPROVED** [review]

**Comment by brockboland** [comment]
I don't have time to review this, I'm just happy to see your names in my inbox.

**Review by joevandeventer — APPROVED** [review]

**Comment by designatednerd** [comment]
Miss you, Brock!

Anyone got a second to merge and push up to CocoaPods if y'all are OK with this? I can't start hammering on the Vokoder stuff until this update is available because of the way dependencies work in CP 😛

**Comment by designatednerd** [comment]
thank you @bryanluby!

**Comment by bryanluby** [comment]
I'm not an owner of this pod so I can't push the pod spec update

**Comment by designatednerd** [comment]
Ironically I may still be. Will give it a shot if @vokalisaac doesn't beat me to it

-ES

Hand-delivered by the Mobile Internet Gnomes

**Comment by vokal-isaac** [comment]
`pod trunk push`'d.

**Pull request merged**